docs/source/http_api/access.rst

```diff
@@ -99,12 +99,21 @@ A simple example in Python::
 
     import hashlib
     import hmac
-    import urllib
     import time
-
-
-    def build_indico_request(path, params, api_key=None, secret_key=None, only_public=False, persistent=False):
-        items = params.items() if hasattr(params, 'items') else list(params)
+    from future.moves.urllib.parse import urlencode
+    ## python 3
+    # from urllib.parse import urlencode
```

Comment:
Why not try importing this and then catch `ImportError` to use the python2 fallback?

Reply:
again, no preference from my side what you want to recommend your users.